spot_rl_experiments/spot_rl/utils/helper_nodes.py

```diff
@@ -59,6 +72,21 @@ def publish_msgs(self):
         # Limit publishing to 10 Hz max
         if time.time() - self.last_publish > 1 / 10:
             self.pub.publish(msg)
+            # publish spot to spotWorld transform as ROS TF
+            self.static_tf_broadcaster.sendTransform(
+                self.spot.get_vision_tform_body_TransformStamped(
+                    robot_kinematic_snapshot_tree
+                )
+            )
+            # publish pose as ROS Odometry
+            msg = Odometry()
+            pose = self.spot.get_vision_T_body_Pose(robot_kinematic_snapshot_tree)
+            msg.pose.pose = pose
+            msg.header.stamp = rospy.Time.now()
+            msg.child_frame_id = "spot"
+            msg.header.frame_id = "spotWorld"
```

Comment:
shouldn't these be constants in Spot class rather than hardcoded strings here?

Reply:
From ros_topics.yaml

Reply:
addressed